script.sql

```diff
@@ -0,0 +1,125 @@
+CREATE TABLE Speciality 
```

Comment:
let's use:
UPPER CASE for SQL keywords (example: SELECT, JOIN, INSERT etc..)
lower_case_with_underscores - names (identifiers) (example: table names, table columns etc..)